tools/js-sdk-release-tools/src/utils/generateInputUtils.ts

```diff
@@ -79,6 +81,11 @@ export async function parseInputJson(inputJson: any) {
     const isMgmtWithHLC = isTypeSpecProject ? false : readmeMd!.includes('resource-manager');
     const isMgmtWithModular = await isManagementPlaneModularClient(specFolder, typespecProjectFolder);
     const sdkType = getSDKType(isMgmtWithHLC, isMgmtWithModular);
+
+    if (apiVersion &&apiVersion.toLowerCase().includes('preview') && sdkReleaseType.toLowerCase() === 'stable') {
```

Comment:
```suggestion
    if (apiVersion && apiVersion.toLowerCase().includes('preview') && sdkReleaseType.toLowerCase() === 'stable') {
```

Reply:
I belive in js the preview version should be `beta`, right? @MaryGao

Reply:
i think there is difference between api-version and sdk version, api-version for preview is `preview` but sdk version for preview is `beta`.